tests/index.ts

```diff
@@ -47,7 +48,7 @@ const conditionalTest = (
   timeout?: number | undefined
 ) => (skipTest ? it.skip(name, fn, timeout) : it(name, fn, timeout));
 
-export type TestCallback = (driver: TrunkDriver) => unknown;
+export type TestCallback = (driver: TrunkLintDriver) => unknown;
```

Comment:
I think it might be useful to have preCheck for more intensive tools tests in the future. We should sync on long-term testing planning